.changeset/fluffy-lies-complain.md

```diff
@@ -0,0 +1,18 @@
+---
+'@backstage/techdocs-common': patch
```

Comment:
With the API interface addition to `PublisherBase`, probably more appropriately a `minor` change, now.